boards/stm32f746g-disco/doc.txt

```diff
@@ -0,0 +1,5 @@
+/**
+ * @defgroup    boards_stm32f746g-disco STM32F746G-DISCO
+ * @ingroup     boards
+ * @brief       Support for the STM32F746G-DISCO board.
+ */
```

Comment:
If the documentation could be beefed up a bit it would be awesome.

Reply:
Added some extra documentation.